Gemfile

```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
```

Comment:
use gem mysql2 instead of sqlite3
after that, you bundle install and restart server

Reply:
regarding the mysql2, I did try to add the mysql to rail file (not the one I push to this repo) by using "rails new appname -d mysql". I also have the ruby gem of mysql2 install. But when I load the localhost,the error occured for mysql2. But later I will try again and test on this file for mysql2 installation

Reply:
done